client/src/components/ProfileSettings/ManageBooking/BookingList/BookingItem.tsx

```diff
@@ -1,37 +1,87 @@
-import { Grid, Avatar } from '@material-ui/core';
+import { useState } from 'react';
+import axios from 'axios';
+import { Grid, Avatar, Menu, Button } from '@material-ui/core';
 import Paper from '@material-ui/core/Paper';
 import Typography from '@material-ui/core/Typography';
 import SettingsIcon from '@material-ui/icons/Settings';
 import useStyles from './useStyles';
-import { useAuth } from '../../../../context/useAuthContext';
-
 interface Props {
   status: string;
-  date: string;
-  imageSrc: string;
-  ownerName: string;
+  bookingDate: string;
+  imgUrl: string;
+  userName: string;
+  startTime: string;
+  endTime: string;
+  category: string;
+  index: number;
 }
 
-const BookingItem = ({ status, date, imageSrc, ownerName }: Props): JSX.Element => {
+const BookingItem = ({
+  status,
+  bookingDate,
+  imgUrl,
+  userName,
+  startTime,
+  endTime,
+  index,
+  category,
+}: Props): JSX.Element => {
   const classes = useStyles();
-  const { loggedInUser } = useAuth();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleAccept = async (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(null);
+    try {
+      await axios.post(
```

Comment:
Can we please define this API call in a function and move it to the helpers folder?